packages/node/src/login/oidc/redirectHandler/AuthCodeRedirectHandler.ts

```diff
@@ -50,8 +50,17 @@ import { KeyObject } from "crypto";
 import { fetch as globalFetch } from "cross-fetch";
 
 import { EventEmitter } from "events";
+import { Headers as NodeHeaders } from "node-fetch";
```

Comment:
And actually, if I'm right, we might not need so much node specific code.
(I think the correct fix here is not importing Headers at all since it's only there to override global.)